wman-backend/Wman.WebAPI/Controllers/CalendarEventController.cs

```diff
@@ -14,9 +14,9 @@ namespace Wman.WebAPI.Controllers
     /// <summary>
     /// CalendarEvent controller
     /// </summary>
-    [Authorize]
     [Route("[controller]")]
     [ApiController]
+    [Authorize]
```

Comment:
You should apply the following `Authorize` attribute in order not to repeat the code at every method. **After updating the attribute, remove the unneccesary attributes at the methods.**
```suggestion
    [Authorize(Roles = "Admin, Manager, Worker")]
```